src/services_models/services_constructor.jl

```diff
@@ -129,7 +128,7 @@ function construct_service!(
             #    throw(IS.ConflictingInputsError("All area most have an AGC service assigned in order to model the System's Frequency regulation"))
         end
     end
-    area_mismatch_variables!(psi_container, areas)
+    add_variables!(AreaMismatchVariable, PSY.Area, psi_container, areas)
```

Comment:
This is where this model kind of breaks a little bit since AreaMismatchVariable is very specific and unique to AGC models. For now we can leave it, but we need to keep in mind how to avoid proliferation of variable types.

Reply:
Would another of the variable types work?  ServiceVariable + PSY.Area may be sufficient for dipatch.